recipes/prophyle/meta.yaml

```diff
@@ -10,7 +10,7 @@ package:
   #sha256:
 
 build:
-  number: 0
+  number: 1
   skip: True # [py27]
   script: pip install --upgrade git+https://github.com/karel-brinda/prophyle
```

Comment:
This is the url I was looking for:
https://pypi.io/packages/source/p/prophyle/prophyle-0.1.0.34.tar.gz
It does not contain any hashes.